src/main/java/core/basesyntax/model/User.java

```diff
@@ -58,4 +64,13 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(login, password, age);
     }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "id=" + id
+                + ", login='" + login + '\''
+                + ", password='" + password
```

Comment:
this is not really secure to show user password even in toString()
keep it in mind for future)